README.md

```diff
@@ -182,6 +189,22 @@ end
 
 Where `your_user_search_function` is a piece of logic in your app that is supposed to find a user, given a user_id. Regardless of whether the user is a new or an existing one, Yoti will always provide their profile, so you don't necessarily need to store it.
 
+You can retrieve the sources and verifiers for each attribute as follows:
+
+```ruby
+given_names_sources = profile.given_names.sources // list or array of anchors
```

Comment:
why can it be a list or an array? Since it is docs, can you please disambiguate this?